k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientPresenter.java

```diff
@@ -277,10 +277,10 @@ public void onPrepareOptionsMenu(Menu menu) {
             menu.findItem(R.id.openpgp_encrypt_enable).setVisible(!isEncrypting);
             menu.findItem(R.id.openpgp_encrypt_disable).setVisible(isEncrypting);
 
-            boolean showSignOnly = K9.getOpenPgpSupportSignOnly();
+            boolean hideSignOnly = account.getPgpHideSignOnly();
```

Comment:
for consistency with the setting in the ui, I guess? I assumed that's what other developers would use as a base for their assumptions about the code, rather than the historical fact that the setting used to be the other way around. I'm good either way though, doesn't really matter

Reply:
For me "hide" is a negative. Using `!` with it makes it a double negative which makes it hard to read. Consistency with settings names is nice, but there are times to deviate. In this case I'd just rename the local variable but keep the name of `Account.getPgpHideSignOnly()`, i.e.

```java
boolean showSignOnly = !account.getPgpHideSignOnly();
```